examples/container-image/main.tf

```diff
@@ -123,6 +123,9 @@ module "docker_build_from_ecr" {
   build_args = {
     FOO = "bar"
   }
+  # Can also use buildx
```

Comment:
Could you please add one more Terraform example showing buildx and multi-stage builds?

Reply:
Done